From 064cd6e590519b6a049d4aa2ee1e0b19feac08ca Mon Sep 17 00:00:00 2001
From: jhenrique09 <jhsv09@gmail.com>
Date: Wed, 26 Oct 2022 22:30:26 -0300
Subject: [PATCH 04/11] WalletActivity: Don't explicity request face auth

There is currently no feedback or indication that biometric authentication is in progress.

The user must explicitly tap the "Unlock to Pay" button for biometrics to be requested.

Change-Id: Ife8a25db1d64a487ed7eaaa9c4959158d3dfd93c
---
 .../src/com/android/systemui/wallet/ui/WalletActivity.java    | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/wallet/ui/WalletActivity.java b/packages/SystemUI/src/com/android/systemui/wallet/ui/WalletActivity.java
index d03148cee335..83def7645c13 100644
--- a/packages/SystemUI/src/com/android/systemui/wallet/ui/WalletActivity.java
+++ b/packages/SystemUI/src/com/android/systemui/wallet/ui/WalletActivity.java
@@ -209,14 +209,14 @@ public class WalletActivity extends ComponentActivity implements
                 true,
                 Utils.getColorAttrDefaultColor(
                         this, com.android.internal.R.attr.colorAccentPrimary));
-        mKeyguardViewManager.requestFace(true);
+        //mKeyguardViewManager.requestFace(true);
     }
 
     @Override
     protected void onPause() {
         super.onPause();
         mKeyguardViewManager.requestFp(false, -1);
-        mKeyguardViewManager.requestFace(false);
+        //mKeyguardViewManager.requestFace(false);
     }
 
     @Override
-- 
2.25.1

